src/test/java/org/openhmis/test/BaseIntegrationTest.java

```diff
@@ -78,21 +50,26 @@ public static void createSchema() throws Exception {
         	properties.getProperty("hibernate.connection.username"),
         	properties.getProperty("hibernate.connection.password"));
         flyway.migrate();
+
     }
 
     @Before
     public void importDataSet() throws Exception {
-//        IDataSet dataSet = getDataSet();
-//        cleanlyInsert(dataSet);
+    	
+    	// Create a clean copy of all data before each test
+        IDataSet dataSet = getDataSet();
```

Comment:
Ah, my speculation that you were going to fill this in later is proved correct.  Mu-wa-ha-ha-ha-haaaaaah.